include/solvers/mf_navier_stokes_operators.h

```diff
@@ -67,7 +68,7 @@ class NavierStokesOperatorBase : public Subscriptor
   NavierStokesOperatorBase(const Mapping<dim>              &mapping,
                            const DoFHandler<dim>           &dof_handler,
                            const AffineConstraints<number> &constraints,
-                           const Quadrature<dim>           &quadrature,
+                           const Quadrature<1>             &quadrature,
```

Comment:
Let's keep a dim-quadrature. This is what you need for simplices.

Reply:
Good to know